pkg/apply/processor/create.go

```diff
@@ -111,29 +108,29 @@ func (c *CreateProcessor) CheckImageType(cluster *v2.Cluster) error {
 
 func (c *CreateProcessor) PreProcess(cluster *v2.Cluster) error {
 	logger.Info("Executing pipeline PreProcess in CreateProcessor.")
-	err := c.RegistryManager.Pull(types.DefaultPlatform(), types.PullPolicyMissing, cluster.Spec.Image...)
+	err := c.Buildah.Pull(buildah.DefaultPlatform(), "missing", cluster.Spec.Image...)
 	if err != nil {
 		return err
 	}
 	if err = c.CheckImageType(cluster); err != nil {
 		return err
 	}
 	for _, img := range cluster.Spec.Image {
-		clusterManifest, err := c.ClusterManager.Create(fmt.Sprintf("%s-%s", cluster.Name, rand.Generator(8)), img)
+		bderInfo, err := c.Buildah.Create(rand.Generator(8), img)
```

Comment:
fmt.Sprintf("%s-%s", cluster.Name, rand.Generator(8)) , It's better to distinguish the cluster name.

Reply:
> fmt.Sprintf("%s-%s", cluster.Name, rand.Generator(8)) , It's better to distinguish the cluster name.

I've move it to `buildah.Interface`, so `ListContainers` function can only filter working containers that belongs to this cluster.